phpstan.neon

```diff
@@ -47,6 +47,17 @@ parameters:
         # Ignore the wrong return type hint of the UrlGeneratorInterface::generate() method
         - '#Method Contao\\CoreBundle\\Picker\\AbstractPickerProvider::generateUrl\(\) never returns null so it can be removed from the return type\.#'
 
+        # Symfony 6.4.0 uses a weird deprecation technique for the UriSigner class that PHPStan cannot handle.
+        - '#Class Symfony\\Component\\HttpKernel\\UriSigner does not have a constructor and must be instantiated without any parameters\.#'
+        - '#Call to an undefined method Symfony\\Component\\HttpKernel\\UriSigner::sign\(\)\.#'
+        - '#Call to an undefined method Symfony\\Component\\HttpKernel\\UriSigner::checkRequest\(\)\.#'
+        - '#Trying to mock an undefined method sign\(\) on class Symfony\\Component\\HttpKernel\\UriSigner\.#'
+        - '#Trying to mock an undefined method checkRequest\(\) on class Symfony\\Component\\HttpKernel\\UriSigner\.#'
+        - '#expects Symfony\\Component\\HttpFoundation\\UriSigner\|null, Symfony\\Component\\HttpKernel\\UriSigner given#'
```

Comment:
Maybe we can remove this in the future if https://github.com/symfony/symfony/pull/52798 gets merged.